source/testing/testing-components.md

```diff
@@ -285,23 +335,25 @@ const locationStub = Service.extend({
   }
 });
 
-moduleForComponent('location-indicator', 'Integration | Component | location indicator', {
-  integration: true,
+module('Integration | Component | location indicator', function(hooks) {
+  setupRenderingTest(hooks);
 
-  beforeEach: function () {
-    this.register('service:location-service', locationStub);
+  hooks.beforeEach(function(assert) {
+    this.owner.register('service:location-service', locationStub);
     // Calling inject puts the service instance in the context of the test,
     // making it accessible as "locationService" within each test
     this.inject.service('location-service', { as: 'locationService' });
```

Comment:
this does not work anymore and is unrelated to the concept of mocking and stubbing anyways